core/internals/internals.py

```diff
@@ -0,0 +1,46 @@
+import pkgutil
+import importlib
+import logging
+
+from mongoengine import *
+
+from core.database import YetiDocument
+from core.constants import DB_VERSION
+from core.constants import MIGRATIONS_DIRECTORY
+
+class Internals(YetiDocument):
+    db_version = IntField(default=0)
+    name = StringField(default="default", unique=True)
+    __internal = None
+
+    @classmethod
+    def syncdb(klass):
+        current_version = klass.get_internals().db_version
+        if DB_VERSION > current_version:
+            print "[+] Database version outdated: {} vs. {}".format(current_version, DB_VERSION)
+            klass.apply_migrations(current_version, DB_VERSION)
+        else:
+            print "[+] Database version is synced with code."
+
+    @classmethod
+    def get_internals(klass):
+        if klass.__internal is None:
+            klass.__internal = Internals.get_or_create(name="default")
```

Comment:
When created, the document should have the current version, because old migrations are not needed.